node-services/cards-external-diffusion/src/domain/server-side/handlebarsHelpers.ts

```diff
@@ -320,6 +322,38 @@ export class HandlebarsHelper {
             return moment().valueOf();
         });
     }
+
+    private static registerDeltaToHtml() {
+        Handlebars.registerHelper('deltaToHtml', (delta) => {
+            const fs = require('fs');
+            let quillFilePath = require.resolve('quill');
+            let quillMinFilePath = quillFilePath.replace('quill.js', 'quill.min.js');
+            
+            let quillLibrary = fs.readFileSync(quillMinFilePath);
+            const TEMPLATE =  `<div id="editor"></div>
+            <script>${quillLibrary}</script>
+            <script>
+              document.getSelection = function() {
+                return {
+                  getRangeAt: function() { }
+                };
+              };
+              document.execCommand = function (command, showUI, value) {
+                try {
+                    return document.execCommand(command, showUI, value);
+                } catch(e) {}
+                return false;
+              };
+            </script>`
+
+            const DOM = new JSDOM(TEMPLATE, { runScripts: 'dangerously', resources: 'usable' });
+            const quill = new DOM.window.Quill('#editor');
```

Comment:
I would load all this once on startup and store quill as static variable .

So you just have to call the setContents in the helper method